### ISSUE TITLE
Add tests to ensure newer JavaScript features are not used

### ISSUE THREAD

**Issue by nickcolley** [alphagov/static#1025]
To ensure this can't happen in the first place: https://github.com/alphagov/static/pull/1021#event-1068537865

**Comment by fofr** [comment]
Should "are not used" be changed to "do not break older browsers"?

Could this be achieved by running jasmine spec runners in older browsers, maybe automating with browserstack or other similar tools?

**Comment by nickcolley** [comment]
Indeed, if we're using newer features we should be polyfilling/transpiling.

We could indeed use those tools to check for errors.
